angular/src/lib/select/select.component.ts

```diff
@@ -1,41 +1,36 @@
 import {
```

Comment:
Just an overall thought for this file...should it now be broken into smaller private files as this is now a 700 line file? @pauljeter